java-manta-client-unshaded/src/main/java/com/joyent/manta/http/EncryptionHttpHelper.java

```diff
@@ -110,17 +110,29 @@
      */
     private final SupportedCipherDetails cipherDetails;
 
+
+    /**
+     * Creates a new instance of the helper class.
+     *  @param connectionContext saved context used between requests to the Manta client
+     * @param config configuration context object
+     */
+    public EncryptionHttpHelper(final MantaConnectionContext connectionContext,
+                                final ConfigContext config) {
+        this(connectionContext, null, config);
```

Comment:
Don't make the non-deprecated constructor call out to the deprecated constructor. Create a new constructor that doesn't use the deprecated code path.

Reply:
This is the only substantial comment I had on the changes and the only reason I asked for changes.